src/content/nft/index.md

```diff
@@ -0,0 +1,195 @@
+
+Fungibile items on the other hand can be exchanged as their value defines them rather than their unique properties. For example ETH is fungible because 1 ETH is exchangeable for another ETH.
+
+### NFT examples
+
+The NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:
+
+- A unique digital artwork.
+- A unique shirt in a limited-run fashion line.
+- An in-game item.
+- [An essay](https://zora.co/0x517bab7661C315C63C6465EEd1b4248e6f7FE183/145).
+- [A digital collectible](https://www.larvalabs.com/cryptopunks/details/1).
+- [A domain name](https://app.ens.domains/name/ethereum.eth)
+- A ticket that gives you access to an event or a coupon – these items are often known as semi-fungible. They are fungible until they are redeemed, at which point they can't be traded for a non-redeemed version.
+
+### ethereum.org examples
+
+#### POAPs (Proof of attendance protocol)
+
+And if you contribute to ethereum.org, you can claim a POAP . These are NFTs that prove you participated in an event. [More on contributing](/contributing/#poap).
+
+![ethereum.org POAP](../../assets/use-cases/poap.png)
+
+#### ethereum.eth
+
+This website is accessible via the ethereum.eth ENS (Ethereum name service) name. With the content hosted on IPFS, a decentralized storage solution, this website is actually an NFT.
+
+- [ENS record](https://app.ens.domains/name/ethereum.eth)
+- [IPFS link](https://ethereum.eth.link/) – _the content hosted on IPFS is currently an older version of this site. So, ethereum.eth is an NFT of a historical version of this site!_
+
+## How do NFTs work?
+
+NFTs have some special properties:
+
+- each token is unique.
+- they're not interchangeable with another NFT. For example 1 ETH is exactly the same as another ETH. This isn't the case with NFTs.
+- each token has an owner and this information is easily verifiable.
+- they live on Ethereum, so are compatible with every single NFT market on Ethereum.
+
+In other words, if you own an NFT:
+
+- only you own it.
+- you can easily prove you own it.
+- no one can manipulate it in any way.
+- you can sell it on, and in some cases this will earn the original creator a % of resale royalties.
+
+And if you create an NFT:
+
+- you can easily prove you're the creator.
+- you determine the scarcity.
+- you can earn royalties every time it's sold.
+- you can sell it on any NFT market or peer-to-peer. You're not locked in to any platform and you don't need anyone to intermediate.
+
+### Fractional ownership
+
+Of course not every single item has a single owner. Songs can have multiple writers, you might own your property with your partner, and you might want to sell a percentage of ownership in your film project to help get it off the ground.
+
+For this reason, NFTs also support multiple owners.
+
+## What are NFTs used for?
```

Comment:
There are differences between ERC720 and ERC1155 – do you think this difference is worth explaining on this page? Or should we just reference it in the build section at the end? Thoughts @samajammin @wackerow ?